### PR TITLE
Add "Course" link and "..." dropdown to nav

### LINKED ISSUE #132: Educational Course Connect
With the new educational course coming out at the end of the quarter, we need to determine how people will access it from our websites.

Each website will access the same school: Iterative Tools School.

[Here is information on how their domains work](https://support.learnworlds.com/support/solutions/articles/12000080095-general-overview-custom-domains#What-is-a-custom-domain?). Let me know how you think it's best to proceed!

@julieg18 @rogermparent @yalozhkin

### PR DESCRIPTION
**Pull request by julieg18**
Fixes #132

### CODE REVIEW

File: src/components/organisms/SiteHeader/index.tsx [hunk 1]
```diff
@@ -123,70 +150,11 @@ const OtherToolsPopup: React.FC<IOtherToolsPopupProps> = ({ list, isOpen }) => {
...
-
-  useEffect(() => {
-    if (!isInstallPopupOpen && !isOtherToolsPopupOpen) {
-      document.removeEventListener('click', handlePageClick)
-      document.removeEventListener('keyup', handlePageKeyup)
-    }
-  }, [isInstallPopupOpen, isOtherToolsPopupOpen])
+  const installPopup = usePopup()
+  const otherToolsPopup = usePopup()
+  const otherPopup = usePopup()
```
Comment (julieg18) [inline src/components/organisms/SiteHeader/index.tsx:157]: Same with other sites, I used a hook to reduce complexity.

File: src/gatsby-plugin-theme-ui/index.js [hunk 2]
```diff
@@ -413,6 +417,23 @@ export default {
               backgroundColor: '#13ADC7'
             }
           }
+        },
+        OtherPopup: {
```
Comment (julieg18) [inline src/gatsby-plugin-theme-ui/index.js:421]: Examples of how it looks: 

image

image

image

**Review by casperdcl — COMMENTED** [review]
AFAIK the course doesn't (yet) have much CML content, but the way the link is currently presented it makes it seem to be a CML course.

wdyt @dmpetrov?

**Comment by julieg18** [comment]
This pr as been open for a while... Should we go ahead and merge this or close it @casperdcl?

**Comment by julieg18** [comment]
This pr has been sitting here for a while now... @casperdcl should I go ahead and merge or close this pr? We do have a "Course" link on all other sites (minus Studio) at the moment 🤔

**Comment by julieg18** [comment]
Closing this I believe we're going to wait till the course site has more cml content on it

**Pull request closed without merging**